server/controllers/index.ts

```diff
@@ -1,6 +1,8 @@
 import type Express from 'express'
```

Comment:
Random thought, we should probably rename this file to `baseController.ts` at some point.

Reply:
hmm dunno. maybe? right now the imports look like `import { BaseController } from '../../controllers'` but is `import { BaseController } from '../../controllers/baseController'` any better?

Reply:
> is import { BaseController } from '../../controllers/baseController' any better?
Almost call for eslint's favourite thing isn't? (the default export) - I don't find this particularly offensive but I like the idea of `BaseController` being in `baseController.ts` as it's more predictable and much easier to find/jump to it in the editor (via CMD+P and "basecon...")

But as I said, it's a bit of random thought and not technically part of this nor a _problem_ - just an observation/preference.